Sources/PathBuilder/LibraryContent.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+import DeveloperToolsSupport
+
+@available(macOS 11.0, *)
+struct LibraryContent: LibraryContentProvider {
+    var views: [LibraryItem] {
+        LibraryItem(
+            Path {
+                Move(to: CGPoint(x: 0, y: 0))
+                // ...
+                Close()
+            }
+        )
+        LibraryItem(
+            Subpath {
+                Move(to: CGPoint(x: 0, y: 0))
+                // ...
+            }
+        )
+        LibraryItem(Arc(center: .zero, radius: 20, startAngle: .zero, endAngle: .degrees(180)))
+        LibraryItem(Close())
+        LibraryItem(Curve(to: CGPoint(x: 10, y: 10), control1: CGPoint(x: 5, y: 5), control2: CGPoint(x: 10, y: 10)))
+        LibraryItem(EmptySubpath())
+        LibraryItem(Line(to: CGPoint(x: 50, y: 50)))
+        LibraryItem(Lines(CGPoint(x: 60, y: 60), CGPoint(x: 70, y: 70)))
+        LibraryItem(Loop(sequence: 0..<10) { i in
+            Line(to: CGPoint(x: i * 5, y: i * 5))
+        })
```

Comment:
Compiler failure in Xcode 14.3.1 is caused by this loop, we can remove it for now.

```suggestion
```

Reply:
Does removing this also fix the issue shown in the snapshot above? Sorry, I never expected it wouldn't work with latest Xcode.

Reply:
Yes, this fixes all the compiler issues.